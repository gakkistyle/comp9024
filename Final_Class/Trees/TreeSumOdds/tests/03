5 2 1 4 7 6 9
1 2 4 5 6 7 9
