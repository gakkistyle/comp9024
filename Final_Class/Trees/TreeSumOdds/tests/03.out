Enter the preorder traversal of the tree: Enter the in-order traversal of the tree: Tree:

      5
     / \
    /   \
   /     \
  2       7
 / \     / \
1   4   6   9

TreeSumOdds returned 22
